recipes/cudadecon/meta.yaml

```
@@ -0,0 +1,101 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "cudaDecon" %}
+{% set version = "0.0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub, remove the line above,
+  # uncomment the line below, and modify as needed. Use releases if available:
+  # url: https://github.com/scopetools/cudaDecon/releases/download/{{ version }}/simplejson-{{ version }}.tar.gz
+  # and otherwise fall back to archive:
+  url: https://github.com/scopetools/cudaDecon/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 29182331bb48d37b415d50365099314af9c824059cd6e9bc687e6d403d699e8e
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
+  number: 0
+  skip: True  # [win or osx]
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: |
+    find .
+    mkdir cmake_build \
+      && cd cmake_build \
+      && cmake ${CMAKE_ARGS} -DCMAKE_BUILD_TYPE=Release ../src \
+      && make \
+      && make install
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - python
+    - cmake
+    - make  # [not win]
+    - ninja  # [win]
+  host:
+    - pip
+    - boost-cpp
+    - libtiff
+    - fftw
+    # - ninja
+    # - cudatoolkit-dev =10.1.243
+    # - cmake
+    # - wget
+  run:
+    # - python
```

Need `python` for Python packages

```suggestion
  run:
    - python
```

---

`pycudadecon` is actually a separate package that will call this `cudadecon` package (and I'll get around to that recipe next).